Automattic/jetpack
# Add Backup Storage info to At a Glance page

#### Changes proposed in this Pull Request:

* This PR adds the a new backup storage details to At a Glance page
* It replace the existing backup card there

#### Jetpack product discussion


HACK WEEK project p6TEKc-61M-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Boot up the branch and run a JT site and connect Jetpack with Free plan
* Goto At a Glance page
* Confirm that the new info graphic is not shown if there are less than or equal to 20 posts.
* Confirm that you see the new info graphic that shows number of posts, plugins and comments, after the site has more than 20 posts.
* Confirm that the horizontal chart shows the exact numbers
* Confirm that the bar chart is blocked by a popup as shown in the screenshot below
* Confirm that clocking the popup hides the info graphic

P.S. I haven't changed the design of the backup card. If we want to do that, we can do it in a separate PR.

<img width="1072" alt="Screenshot 2022-03-11 at 5 24 41 PM" src="https://user-images.githubusercontent.com/18226415/157868395-9244eff9-d047-4417-9050-b0f754c55f77.png">
